java/spacewalk-java.changes

```diff
@@ -1,3 +1,5 @@
+handle centos urls that contain repo target in query string (bsc#1171996)
```

Comment:
This was incorrect, as it's missing the `-` to mark the start of the entry. Will fix it during tagging.